x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/reindex/flyout/__snapshots__/warning_step.test.tsx.snap

```diff
@@ -7,129 +7,114 @@ exports[`WarningsFlyoutStep renders 1`] = `
+            Object {
+              "allField": <EuiCode>
+                _all
+              </EuiCode>,
+            }
+          }
+        />
+      }
+      documentationUrl="https://www.elastic.co/guide/en/elasticsearch/reference/6.0/breaking_60_mappings_changes.html#_the_literal__all_literal_meta_field_is_now_disabled_by_default"
+      label={
+        <FormattedMessage
+          defaultMessage="{allField} will be removed"
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.warningsStep.allFieldWarningTitle"
+          values={
+            Object {
+              "allField": <EuiCode>
+                _all
+              </EuiCode>,
+            }
+          }
+        />
+      }
+      onChange={[Function]}
+      warning={0}
     />
-    <EuiText
-      grow={true}
-      size="m"
-    >
-      <EuiCheckbox
-        checked={false}
-        compressed={false}
-        disabled={false}
-        id="reindexWarning-1"
-        indeterminate={false}
-        label={
-          <strong>
-            Boolean data in 
-            <EuiCode>
-              _source
-            </EuiCode>
-             might change
-          </strong>
+    <WarningCheckbox
+      checkedIds={
+        Object {
+          "reindexWarning-0": false,
+          "reindexWarning-1": false,
         }
-        onChange={[Function]}
-      />
-      <p
-        className="upgWarningsStep__warningDescription"
-      >
-        If a documents contain a boolean field that is neither 
-        <EuiCode>
-          true
-        </EuiCode>
-         or
-         
-        <EuiCode>
-          false
-        </EuiCode>
-         (for example, 
-        <EuiCode>
-          "yes"
-        </EuiCode>
-        ,
-         
-        <EuiCode>
-          "on"
-        </EuiCode>
-        , 
-        <EuiCode>
-          1
-        </EuiCode>
-        ), reindexing converts these fields to 
-        <EuiCode>
-          true
-        </EuiCode>
-         or 
-        <EuiCode>
-          false
-        </EuiCode>
-        . Ensure that no application code or scripts rely on boolean fields in the deprecated format.
-        <br />
-        <EuiLink
-          color="primary"
-          href="https://www.elastic.co/guide/en/elasticsearch/reference/6.0/breaking_60_mappings_changes.html#_coercion_of_boolean_fields"
-          target="_blank"
-          type="button"
-        >
-          Documentation
-        </EuiLink>
-      </p>
-    </EuiText>
-    <EuiSpacer
-      size="l"
+      }
+      description={
+        <FormattedMessage
+          defaultMessage="If a document contain a boolean field that is neither {true} or {false}
+                   (for example, {yes}, {on}, {one}), reindexing converts these fields to {true} or {false}.
+                   Ensure that no application code or scripts rely on boolean fields in the deprecated format."
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.warningsStep.apmReindexWarningDetail"
```

Comment:
Looks like this should be `xpack.upgradeAssistant.checkupTab.reindexing.flyout.warningsStep.booleanFieldsWarningDetail`

Reply:
Fixed!